src/Psapi/Psapi.cs

```diff
@@ -34,5 +35,42 @@ public static partial class Psapi
         [DllImport(nameof(Psapi), SetLastError = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
         public static extern bool EmptyWorkingSet(SafeObjectHandle hProcess);
+
+        /// <summary>
+        /// Retrieves a handle for each module in the specified process that meets the specified filter criteria.
+        /// </summary>
+        /// <param name="hProcess">A handle to the process.</param>
+        /// <param name="lphModule">An array that receives the list of module handles.</param>
+        /// <param name="cb">The size of the lphModule array, in bytes.</param>
+        /// <param name="lpcbNeeded">The number of bytes required to store all module handles in the lphModule array.</param>
+        /// <param name="dwFilterFlag">The filter criteria. This parameter can be one of the following values.</param>
+        /// <returns>If the function succeeds, the return value is nonzero. If the function fails, the return value is zero. To get extended error information, call GetLastError.</returns>
+        [DllImport(nameof(Psapi), SetLastError = true)]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        public static extern bool EnumProcessModulesEx(IntPtr hProcess, [MarshalAs(UnmanagedType.LPArray)] [In][Out] IntPtr[] lphModule, uint cb, [MarshalAs(UnmanagedType.U4)] out uint lpcbNeeded, uint dwFilterFlag);
```

Comment:
`cb` should be `int` instead of `uint`, based on our pattern of preferring a .NET friendly type to avoid unnecessary casting for users.
`lphModule` should be `IntPtr*` with a `[Friendly]` attribute with the appropriate flags so that a friendly overload can be generated.
`dwFilterFlag` should be typed as an enum.